src/app/main.cpp

```diff
@@ -113,6 +113,28 @@ typedef SInt32 SRefCon;
 #include "qgsopenclutils.h"
 #endif
 
+/**
+ * Print QGIS version
+ */
+void version( )
+{
+  QStringList msg;
```

Comment:
There doesn't seem to be any benefit of using a list here and joining after. How about

    const QString msg = QStringLiteral("QGIS %1 '%2' (%3)\n").arg( VERSION ).arg( RELEASE_NAME ).arg( QGSVERSION );

Reply:
Much better, thank you. Updated.